internal/action/workspace.go

```diff
@@ -211,23 +212,26 @@ type NewWorkspaceConfigOptions struct {
 	Providers                []Provider
 }
 
+func NewModule() *tfconfig.Module {
+	return &tfconfig.Module{
+		Terraform: tfconfig.Terraform{},
```

Comment:
This is a zero value, it will be equivalent if unset. In general, you should provide a New func for an object when the zero value is not usable and only set the necessary fields.

Reply:
reasonable, will follow up